config/webpack.config.dev.js

```diff
@@ -11,7 +11,7 @@ var path = require('path');
 var autoprefixer = require('autoprefixer');
 var webpack = require('webpack');
 var HtmlWebpackPlugin = require('html-webpack-plugin');
-var paths = require('./paths');
+var paths = require('./')('paths');
```

Comment:
I’m worried that this convention is going to look super confusing to people after ejecting (or even for those who want to contribute to this project). What alternative solutions have you considered?